Misc/NEWS.d/next/Library/2024-09-06-01-35-11.gh-issue-123756.Ozbhke.rst

```diff
@@ -0,0 +1 @@
+Only allow command line usage of :mod:`pdb` to use ``restart`` command.
```

Comment:
News will need to mention the new arg to Pdb as well.

Reply:
How about just "origin"?

Reply:
`PdbOrigin` or `PdbInvocationOrigin`? Yeah I also thought about origin and I think that's a good name.